### PR TITLE
pwd, echo, exit, rm, which: add Russian translation

### PR DESCRIPTION
**Pull request by egor1wtf**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ru/common/echo.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`echo "{{Привет, мир}}"`
+
+- Отобразить сообщение с переменной окружения:
+
+`echo "{{Мой путь - $PATH}}"`
+
+- Отобразить сообщение не перенося каретку на новую строку:
+
+`echo -n "{{Hello World}}"`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/echo.md:16]:
```suggestion
- Отобразить сообщение не перенося каретку на новую строку:

`echo -n "{{Привет, мир}}"`
```

Optional fix: broken consistency: translated placeholder content expected.

File: pages.ru/common/echo.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Добавить сообщение в файл:
+
+`echo "{{Hello World}}" >> {{file.txt}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/echo.md:20]:
```suggestion
- Добавить сообщение в файл:

`echo "{{Привет, мир}}" >> {{путь/к/файлу.txt}}`
```

Almost the same issue as above.
Mandatory fix (error source: original page): [broken path](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): placeholder content conforming `path/to/*` glob expected.

File: pages.ru/common/pwd.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# pwd
+
+> Print name of current/working directory.
+> More information: <https://www.gnu.org/software/coreutils/pwd>.
+
+- Print the current directory:
+
+`pwd`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/pwd.md:8]:
```suggestion
- Распечатать текущий каталог:

`pwd`
```

Optional fix: broken description: translated sentence expected.

File: pages.ru/common/pwd.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+- Print the current directory, and resolve all symlinks (i.e. show the "physical" path):
+
+`pwd -P`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/pwd.md:12]:
```suggestion
- Распечатать текущий каталог и разрешить все симлинки (т.е. показать "физический" путь):

`pwd -P`
```

The same issue as above.

File: pages.ru/common/rm.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# rm
+
+> Удалить файлы или каталоги.
+> Больше информации: <https://www.gnu.org/software/coreutils/rm>.
+
+- Удалить файлы из определённых мест:
+
+`rm {{путь/до/файла}} {{путь/до/другого/файла}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/rm.md:8]:
```suggestion
- Удалить файлы из определённых мест:

`rm {{путь/до/файла}} {{путь/до/другого_файла}}`
```

Mandatory fix (error source: original page): [broken path](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): placeholder content conforming `path/to/[^/]*` regex expected.

File: pages.ru/common/rm.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Рекурсивно удалить каталог и все его подкаталоги:
+
+`rm -r {{путь/до/директории}}`
+
+- Принудительно удалить каталог без запроса подтверждения или отображения сообщений об ошибках:
+
+`rm -rf {{путь/до/директории}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/rm.md:16]:
```suggestion
- Принудительно удалить каталог без запроса подтверждения или отображения сообщений об ошибках:

`rm -r -f {{путь/до/директории}}`
```

Optional fix (error source: original page): other reason: joining short options is unpredictable in a general case.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

File: pages.ru/common/rm.md (hunk 6 above)
Comment (EmilySeville7cfg) [reply pages.ru/common/rm.md:16]: [This](https://github.com/tldr-pages/tldr-json-schemas) project will be used to solve this issue.

File: pages.ru/common/echo.md (hunk 1 above)
Comment (egor1wtf) [reply pages.ru/common/echo.md:16]: @EmilySeville7cfg thank you, 
I found another mistake - the lack of a comma.

```suggestion
- Отобразить сообщение, не перенося каретку на новую строку:

`echo -n "{{Привет, мир}}"`
```

Comment (EmilySeville7cfg) [reply pages.ru/common/echo.md:16]: Thanks :)

File: pages.ru/common/pwd.md (hunk 4 above)
Comment (egor1wtf) [inline pages.ru/common/pwd.md:12]: @EmilySeville7cfg 

I apologize for the inattention.

Here is the translation that I wanted to offer. I will pay attention to the translation of the word "Print". You were asked to "Распечатать", this word does not accurately convey the meaning. The word "Распечатать" in Russian usually means printing with a printer. I think it's better to use the word "Отобразить" in English "Display", i.e. output information to the terminal.

```suggestion
# pwd

> Отобразить абсолютной путь до текущей/рабочей директории.
> Больше информации: <https://www.gnu.org/software/coreutils/pwd>.

- Отобразить абсолютной путь до текущей директории:

`pwd`

- Отобразить абсолютной путь до текущей директории и "разрешить" символические ссылки (т.е. показать "фактический" путь):

```

Comment (EmilySeville7cfg) [reply pages.ru/common/pwd.md:12]: Your suggestion is better than Google Translate shown me. :)

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages.ru/common/pwd.md (hunk 4 above)
Comment (egor1wtf) [reply pages.ru/common/pwd.md:12]: @EmilySeville7cfg 

Thank you for the review!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**